### PR TITLE
Project3

### PR DESCRIPTION
**Pull request by mgunduz1**
Newsweek clone project Milestone 1.

We have added the requested Navbar and Footer for Milestones 1. It includes the logos, icons, colors, and complies with responsiveness for phone, tablet, and desktop sizes.

### CODE REVIEW

**Comment by uryelah** [comment]
## Invalid Pull Request 🛑 

I'm sorry to inform I'll have to make you submission as **Invalid** for the following reasons:

1. The Pull Request does not have a relevant description that describes well enough the features achieved in this milestone.

Please review the [Professional GitHub repository lesson](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) and update your  PR message according to the models/links under the lesson *3 basic rules to follow in Microverse projects*.

pr_message


When the fixes are made please re-submit the review for Milestone 1.

**Comment by sergiocortessat** [comment]
Initial pull request modified for proper description milestone 1. "Newsweek clone project. We have added the requested Navbar and Footer for milestones one. It includes the logos, icons, colors, and complies with responsiveness for phone, tablet, and desktop sizes. "

**Review by SpaYco — APPROVED** [review]
# Milestone Approved ✔️ 

Milestone approved, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone.

- [x] you forgot to make some of the elements as links, this includes the logo, Login, Subscribe, footer links, and Social media Icons ([Link to page](https://www.newsweek.com/))


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by sergiocortessat** [comment]
Milestone 2 delivery. We have added the middle section that includes 3 columns. We have added the responsive design for the 3 devices and taken care of the order while resizing.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,697 @@
...
+        <p class="pl-1">The Biden Administration will issue an order stopping construction of the border wall on its first day.
+        </p>
+      </div>
+    </div>
+
+  </section>
+
+<br>
+
+<section style="font-family: Arial, Helvetica, sans-serif;"
```
Comment (SpaYco) [inline index.html]: - [x] Please don't include inline stylings, instead put all of this in the CSS file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,697 @@
...
+<ul class="list-unstyled row col-md-4 m-0 mb-3 justify-content-md-end">
+  <li><i class="fab fa-facebook-square mx-1 fa-2x text-muted"></i></li>
+  <li><i class="fab fa-twitter mx-1 fa-2x text-muted"></i></li>
+  <li><i class="fab fa-instagram mx-1 fa-2x text-muted"></i></li>
+  <li><i class="fab fa-tumblr mx-1 fa-2x text-muted"></i></li>
+  <li><i class="fab fa-linkedin-in mx-1 fa-2x text-muted"></i></li>
+</ul>
+</section>
+
+<footer style="font-family: Arial, Helvetica, sans-serif;" class="pt-5 bg-white row">
```
Comment (SpaYco) [inline index.html]: - [x] Please don't include inline stylings, instead put all of this in the CSS file.

**Review by SpaYco — APPROVED** [review]
# Milestone Approved ✔️ 

Milestone approved, great work on managing to build the webpage without using custom CSS, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone.



**Desktop**
- [x] you're missing the **LATEST NEWS** section in the middle column under the **MORE STORIES** section

**Tablet**
- [x] you're missing the **LATEST NEWS** section in the right column under the **MORE STORIES** section

**Mobile**

- [x] you're missing the **LATEST NEWS** section under the **MORE STORIES** section
- [x] there are more **SPONSORED INSIGHT** after the Sign Up section.

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by mgunduz1** [comment]
LATEST NEWS and SPONSORED INSIGHT parts added from the last milestone.

All the requirements for milestone 3 added with responsive design approach.

This is the final version of our bootstrap project.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1564 @@
...
+          <h4 class="font-weight-bold">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h4>
+          <p class="font-weight-lighter">All Access + Weekly Delivery.
+            Choose A Membership That's Perfect for You!</p>
+          <button type="submit" class="btn border-0 outline-none bg-danger text-white p-2 h5 rounded">SUBSCRIBE
+            &gt;</button>
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-4 mt-md-5 my-5 m-lg-0 ">
+      <h6 class="red text-danger">NEWSLETTER</h6>
```
Comment (uryelah) [inline index.html]: - [x] Make sure that this title has the correct font-weight. 
This, 'Subscribe', ' Featured Stories', 'Top Story', 'The Debate', and 'Opinion' in special also have the same issue of having a too low font-weight.

Compare your clone and the original archived site and make sure the text styles match.

Comment (uryelah) [reply index.html]: - [x] Spend more time on the responsiveness of the 'Subscribe' section text on the tablet size:
**Your version**
Screenshot_2021-01-27 Newsweek(3)
**Original version**
subscribe_and_newsletter_tablet

---

- [x] Spend more time on the responsiveness of the 'Newsletter' section image on the mobile size:
**Your version**
Screenshot_2021-01-27 Newsweek
**Original version**
subscribe_and_newsletter_mobile

**Review by uryelah — CHANGES_REQUESTED** [review]
Hi **Mert Gündüz** & **Sergio Cortes Satizabal** , I'm Sarah your reviewer for this PR 👩‍🚀 

You guys did an great job in this milestone! You did specially great in the main section scroll behavior and going beyond what was asking by adding the little favicon 'N' icon to the head of your clone.

There are still some points to work on before finishing this project so just check my feedback carefully, **make the required changes, and then request a re-review for this milestone**!

Remember sure to use the same PR, and to check the requests inside and under this message very carefully 😉 

You are really close so keep on coding and Rock on! 🎸 👨‍🎤 

# Required Changes 🏗️ 🐛 💡

-----

 **Kindly fix the pointed out items and comments below, then start working on the next milestone**
 
- [x] Each article image should also be a link, the same way the titles are. The user of the NewsWeek site is able to click an image and be linked to the article, so make sure there are links around each article image.

---

- [x] Make sure that in the places you use the square Newsweek 'N' logo the logo is not distorted, it should be square and not a thin rectangle.
See how in the screenshots below the 'N' is distorted:

Screenshot_2021-01-27 Newsweek(2)
Screenshot_2021-01-27 Newsweek(5)

*How it should be:*
Screenshot_2021-01-27 Newsweek(1)

---

- [x] Let's make sure the background, the sides of the screen, in the desktop size is the same grey the original site uses instead of just white. Careful to make sure that doesn't show up on the tablet and mobile sizes.

Note how in the screenshot from the original site below you can see the left and right margins of the site with that grey color:
Screenshot_2021-01-27 Newsweek(4)

----

- [x] Make sure  that you are **using the correct shade of red in your clone**, using Bootstrap well is great! But sadly the Bootstrap red is not the same as the Newsweek brand red.

Go over all the places you are using the incorrect shade of red and modify it so it matches the original one.

---

- [ ] [OPTIONAL] If you want to be even more semantic, use the `article` tag for each article inside your clone sections. Read more on the [article tag here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by sergiocortessat** [comment]
Milestone 3: TSE changes requested. Newsletter margins, logo margins, Red color, General margins, Bold titles.

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] You still need to fix the Subscribe / Newsletter section problems mentioned by the previous TSE

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by mgunduz1** [comment]
> # Changes Requested
> Hello Team! 
> Awesome job so far!!  you still need to make some changes though!
> 
> ## Required Changes 
> * [x]  You still need to fix the Subscribe / Newsletter section problems mentioned by the previous TSE
> 
> ### please make the appropriate changes and ask for another code review 
> Happy Coding!
> 
> Best,
> Aziz Mejri
> _contact me on [twitter](https://twitter.com/iSpaYco) or slack (@Spayco) if you need any help_

Hi Aziz, We already did the changes that requested. So we can't understand what you meant with **"You still need to fix the Subscribe / Newsletter section problems mentioned by the previous TSE"**. 

Screenshot_1
Screenshot_2
Screenshot_3

( CURRENT VERSİON OF OUR PROJECT)

**Comment by SpaYco** [comment]
hey @mgunduz1, ~for the subscribe section, it breaks on 650px width~ **(the original website looks like the picture below, so this isn't needed, sorry for the inconvienced)**
image


and the newsletter, the section where it has text and form should be centered

**Comment by sergiocortessat** [comment]
Adding TSE changes request for fix the Subscribe / Newsletter section. We have added the proper format for the screen sizes. Mobile and tablet maintain on the row (email + signup button) and Desktop switch a column style. Thanks

**Review by uryelah — APPROVED** [review]
Hi **Mert Gündüz & Sergio Cortes Satizabal**, I'm Sarah your reviewer for this PR 👩‍🚀 

You did a great job updating your project according to the change requests , good work specially with that last pesky one!

Good luck with your HTML&CSS Capstones! 🚀  

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**

---------

### For your next projects
- [ ] Make sure your next commits follow the best committing formatting practices.
They should all be written in the imperative present mood and be capitalized:
```
// Good
Add first commit

// Bad
adding first commit
```

---------

    
    
**Keep being awesome devs and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**